### DISCUSSION TITLE
update on dolphin stranding + began looking into "Implementing and extend unsupervised human-human text/audio translation"

### DISCUSSION THREAD

**Discussion by radekosmulski** [earthspecies/project#20] [General]
I am continuing with the [dolphin project](https://github.com/earthspecies/dolphin/blob/master/why_do_strandings_occur%3F.ipynb) to familiarize myself with the data and to bring some closure to the work I started. Listened to a bunch of audio from this dataset today. I'm thinking this might be quite a challenging dataset. If we want to test drive our detector in the wild and have a positive conservation impact, I am wondering if it not might be better to focus on one of the projects from Christos? I think there is a chance they could pose a more approachable problem and could be a better testing grounds for our detector. Here, on the dolphin project, we are stacking hypotheses on top of hypotheses (strandings are reflected in dolphin vocalizations -> the difference - even if subtle - can be picked up by our current tools -> per day labeling is sufficient, etc). Or maybe a better way to approach the project would be to just focus on the detector and skip the classification for now? That would be a task amenable to our tooling and automating detections, even if roughly, could add genuine value to the research group.

Further to that, I started to research and document "Implementing and extend unsupervised human-human text/audio translation" under a project [here](https://github.com/orgs/earthspecies/projects/4) - taking inspiration from @kzacarian's awesome approach! 🙂 I went ahead and made the project public but maybe that is too early. I am trying to organize the work so that papers / considerations / steps are ordered from top to bottom, top being more important to my mind, or to be done first.

**Comment by aza** [comment]
If the dolphin stranding project is too far off our roadmap, then my hunch follows yours Radek: we should follow a different project that continues to build our capacity and efficiency and gets us closer to goal. 

I like the bias to start in the open!

**Comment by kzacarian** [comment]
@radekosmulski  - What a fine-looking [project](https://github.com/orgs/earthspecies/projects/4
)!!!
